sources/packages/api/src/route-controllers/student/models/student.dto.ts

```diff
@@ -62,3 +64,29 @@ export interface StudentRestrictionDTO {
   hasProvincialRestriction: boolean;
   restrictionMessage: string;
 }
+/**
+ * DTO object for student details.
+ */
+export interface StudentDetailDTO {
+  firstName: string;
+  lastName: string;
+  email: string;
+  gender: string;
+  dateOfBirth: Date;
```

Comment:
@ann-aot, it is supposed to work either, unless we figure out something that tell us otherwise 😄 
@dheepak-aot, please ensure that the value being displayed reflects the DB date.

Reply:
Yeah I tested for this specific thing. It looks good.